packages/ia-zendesk-help-widget/src/ia-zendesk-help-widget.js

```diff
@@ -0,0 +1,148 @@
+import { html, css, LitElement } from 'lit-element';
+import { LazyLoaderService } from '@internetarchive/lazy-loader-service';
+import loaderIcon from './assets/icon-loader';
+import questionIcon from './assets/icon-question';
+
+class ZenDeskHelpWidget extends LitElement {
+  static get properties() {
+    return {
+      widgetSrc: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    this.widgetSrc = '';
+    this.buttonVisible = true;
+    this.isLoading = false;
+  }
+
+  async initiateZenDesk() {
+    this.isLoading = true;
+
+    // force UI update to render loader icon after button click
+    this.requestUpdate();
```

Comment:
You shouldn't have to force an update like this. If you make `isLoading` reactive, it will update automatically.